### PR TITLE
Update google-cloud-platform-source.md

### PR DESCRIPTION
**Pull request by eli-jang**
resize image size for GCP-advanced-options.png

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by stacykor — CHANGES_REQUESTED** [review]
Hey @eli-jang GitHub markdown doesn't support image widths in this way. 

You can use straight up HTML to achieve this:

File: docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ This Source will be a Google Pub/Sub-only Source, which means that it will only
 
 1. **Advanced Options for Logs**.
 
-  ![GCP advanced options Jan 12 22.png](/img/send-data/GCP-advanced-options.png)
+  ![GCP advanced options Jan 12 22.png](/img/send-data/GCP-advanced-options.png|width=70)
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/google-source/google-cloud-platform-source.md:48]: Needs to be in HTML. So it would be:

`GCP advanced options`

**Review by kimsauce — CHANGES_REQUESTED** [review]
one change

**Comment by eli-jang** [comment]
let me reopen with a new pull request.

**Pull request closed without merging**